packages/runtime-common/module-syntax.ts

```diff
@@ -27,12 +28,11 @@ import type { types as t } from '@babel/core';
 import type { NodePath } from '@babel/traverse';
 import type { FieldType } from 'https://cardstack.com/base/card-api';
 
-export type { PossibleCardClass, Export };
-import { unionWith } from 'lodash';
+export type { PossibleCardClass, BaseDeclaration };
 
 export class ModuleSyntax {
   declare possibleCards: PossibleCardClass[];
```

Comment:
Shall we actually rename this to `possibleCardOrFieldClass`?

Reply:
good idea